backend/de.metas.serviceprovider/src/main/java/de/metas/serviceprovider/github/GithubImporterConstants.java

```diff
@@ -50,4 +47,18 @@
 
 		private final String name;
 	}
+
+	@AllArgsConstructor
+	@Getter
+	enum LabelType{
```

Comment:
do we have a test which, at least indirectly, uses LabelType enum?
reason: make sure those Patterns are compiled without exceptions.

Reply:
yes, we do : `GithubImporterServiceTest.importIssue()`